src/ParseServer.js

```diff
@@ -183,6 +183,8 @@ class ParseServer {
     api.use(bodyParser.json({ type: '*/*', limit: maxUploadSize }));
     api.use(middlewares.allowMethodOverride);
     api.use(middlewares.handleParseHeaders);
+    middlewares.handleRateLimit(api, rateLimit);
```

Comment:
This is middleware always initialized / traversed? Maybe we can save resources and only enable this if the rateLimit option is set?

Reply:
This calls a function which loops through rate limit and adds to express - if ratelimit.length === 0, no middleware will be added

Reply:
Got it.